docs/source/guides/scheduling.rst

```diff
@@ -0,0 +1,127 @@
+============================
+Resource Scheduling Strategy
+============================
+
+To make the best use of machine resources, BentoML provides a few configuration options to control how many runner workers
+will be spawned and how they distribute on the CPUs and GPUs. It can be configured for all runners as well as individual runner.
+The following configuration options are available:
+
+.. tab-set::
+
+    .. tab-item:: All Runners
+       :sync: all_runners
+
+       .. code-block:: yaml
+          :caption: ⚙️ `configuration.yml`
+
+          runners:
+            resources:
+              cpu: 4
+              gpu: 2
```

Comment:
Is this the new configuration field we are introducing? iirc this was `nvidia.com/gpu` b4

Reply:
No, apparently :P. Fixed. Thanks